### ISSUE TITLE
the documentation site is not available

### ISSUE THREAD

**Issue by wenbinyu-PU** [wenbinyugroup/iVABS#57]
I just found out that the documentation site is not available. Do you know why? @unitasium @duhd1993 Is it because I changed the name to iVABS? I changed it back and could not get the documentation site back either.

**Comment by duhd1993** [comment]
It's case-sensitive. This might be why I initially used ivabs. Reason why it still doesn't work when you change it back might be just delay.
https://wenbinyugroup.github.io/iVABS/
We can also consider using a subdomain of analyswift.

**Comment by wenbinyu-PU** [comment]
Great, it is available now. What is the advantage of using a subdomain of AnalySwift?

**Comment by duhd1993** [comment]
I feel it's not very intuitive to change to uppercase when you type a url. The domain part is case-insensitive.

**Comment by wenbinyu-PU** [comment]
Yes, however, usually people don't type a url. Often we can lick the link. Rarely we type. My concern is that the ranking we build up with ivabs is not lost. We need to accumulate again. Is there a way to redirect https://wenbinyugroup.github.io/ivabs/ also to https://wenbinyugroup.github.io/iVABS/?

Another issue is that we have iVABS, a VABS-based Design Framework - iVABS 0.8 documentation<https://wenbinyugroup.github.io/iVABS/>

Do we really need to attach - iVABS 0.8 documentation, or this part can be deleted. Just keep iVABS, a VABS-based Design Framework in the title?

**Comment by wenbinyu-PU** [comment]
One more related issue is that the link under the repository does not work. I tried to change the readme file. It is still not working.

**Comment by duhd1993** [comment]
> One more related issue is that the link under the repository does not work. I tried to change the readme file. It is still not working.

This is because the automatic build is set to push to gh-pages branch. But Su has configured the site to gh-pages-new branch. I don't know if he added something different. It seems the main branch might be out of date. We need confirmation from him.

> Is there a way to redirect https://wenbinyugroup.github.io/ivabs/ also to https://wenbinyugroup.github.io/iVABS/?

Yes. but we need to create a ivabs repo and set up a redirect from there. We can keep it for some time until google recognize the new site. Currently google points to the old one and it leads to a 404 error.

**Comment by wenbinyu-PU** [comment]
Is the re-direction automatic? If yes, please create the new repo to avoid the 404 error.

**Comment by duhd1993** [comment]
It's not very automatic but can be done https://finisky.github.io/en/migrate-github-pages-by-301-redirects/

**Comment by wenbinyu-PU** [comment]
If it is not too much work, please get it done.

**Comment by wenbinyu-PU** [comment]
1. Ok, I have change it back. 
2. The main concern here is that iVABS was mainly developed by Purdue. Just emailed Allan.

**Comment by wenbinyu-PU** [comment]
@unitasium @duhd1993 after careful consideration, I think we can keep ivabs as the repo name according what Su suggested. Then we don't need a subdomain. 

However, sometimes through google, I found the version number attached to a page see below picture. Is it automatic? I will suggest no version number attached to it to keep consistency and have the same page better remembered/ranked by search engine. 
![image](https://github.com/wenbinyugroup/ivabs/assets/78181625/c29e8060-ab12-43cf-8d67-a205851b900c)
